files/homeassistant/hacs_version/individual_ams_lite.yaml

```diff
@@ -46,12 +47,13 @@ elements:
     tap_action:
       action: none
     style:
-      top: 42%
-      left: 15.4%
+      top: 40%
+      left: 45%
       text-align: center
       font-size: 1em
-      background-color: rgba(0,0,0,0.4)
-      box-shadow: 0 0 5px 5px rgba(0, 0, 0, 0.4)
+      background-color: var(--tray_1_bg)
```

Comment:
Preferably I would like to keep the labels background-color consistent as `rgba(0,0,0,0.5)` regardless if it's active or not. Helps with readability. Applies to all 4 labels.

Reply:
My first iteration:

```yaml
background-color: var(--tray_2_bg)
padding: 7px
border: 2px solid rgba(255,255,255,0.6)
```

![image](https://github.com/WolfwithSword/Bambu-HomeAssistant-Flows/assets/38808827/7b04c4d4-17b4-48a5-8741-85fd430a2b79)

What I think could really improve readability (also fixed the alpha-channel consistency)

```yaml
background-color: rgba(0,0,0,0.5)
padding: 7px
border: 2px solid rgba(255,255,255,0.6)
box-shadow: 0 0 5px 5px var(--tray_#_bg)
```

![image](https://github.com/WolfwithSword/Bambu-HomeAssistant-Flows/assets/38808827/de4d77b9-b2ff-400a-9bbc-331837c5af80)

I really don't like the black shadow when the filament is inactive, sorry 😄

Reply:
Oh no worries, I agree for the AMS lite it makes sense to have the hard-border since it's more in the open (with the AMS normal, it was to blend in with the cover). I was initially suggesting to just colour the inside of the filament one and leave the text one alone.

I guess with lighter colour filaments then colouring inside might also be an issue, so yeah I agree - your new version is much nicer! I think because the AMS Lite is so much more "open" it is best to also light up the test too like this now, makes it more visible.